contrib/smtp_proxy/filters/network/source/config.cc

```diff
@@ -0,0 +1,35 @@
+#include "contrib/smtp_proxy/filters/network/source/config.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace SmtpProxy {
+
+/**
+ * Config registration for the SMTP Proxy filter. @see NamedNetworkFilterConfigFactory.
+ */
+Network::FilterFactoryCb
+NetworkFilters::SmtpProxy::SmtpConfigFactory::createFilterFactoryFromProtoTyped(
+    const envoy::extensions::filters::network::smtp_proxy::v3alpha::SmtpProxy& proto_config,
+    Server::Configuration::FactoryContext& context) {
+  ASSERT(!proto_config.stat_prefix().empty());
```

Comment:
I think this ASSERT is unnecessary? This condition should be ensured by the proto validation.

Reply:
Removed it.